src/jit/gentree.cpp

```diff
@@ -11955,6 +11955,10 @@ void Compiler::gtDispLIRNode(GenTree* node)
 
         // 49 spaces for alignment
         printf("%-49s", "");
+#ifdef FEATURE_SET_FLAGS
```

Comment:
@pgavlin `#ifdef` should be `#if` here.
BTW it turns out `FEATURE_...` macro variables are used in two different ways across the code base: some of them are merely defined and get checked by `#ifdef` or `#if defined(...)`, but some are defined to some boolean value and checked by `#if`. That's what I didn't pay attention for.